CHANGELOG.rst

```diff
@@ -1,5 +1,42 @@
 Changelog
 =========
+1.19.1.dev0 (2021-08-25)
+------------------------
+MLflow 1.19.1.dev0 includes several major features and improvements:
+
+Features:
+
+- MLflow UI updates (#4712, @dbczumar)
+- Support distributed pytorch lightning autologging execution (#4717, @dbczumar)
+- Added more Model Registry Functions to R (#4527, @bramrodenburg)
+- Pyspark ml autolog post-training metric (#4686, @WeichenXu123)
+- Scoring Server - Added support for wrapped prediction responses (#4611, @Ark-kun)
+- make sklearn autologging supporting delegated methods in estimators (#4628, @WeichenXu123)
+- Pyspark CrossValidator autolog "std metrics" (#4672, @WeichenXu123)
+- Implement auto pip dependency inference for `mlflow.*.log_model` (#4518, @harupy)
```

Comment:
@harupy Can you write a more descriptive release note for this?

Reply:
```
`mlflow.*.log_model` and `mlflow.*.save_model` now automatically infer the pip requirements of the model to log/save. 
```

Reply:
@harupy Awesome! Can we indicate that the dependency inference is based on the current software environment?